src/utils/renderSchema.ts

```diff
@@ -118,7 +119,7 @@ export const renderSchema: Walker = function*(schema, level = 0, meta = { path:
 
           yield* renderSchema(property, level + 1, {
             ...(i !== 0 && { divider: DIVIDERS[node.combiner] }),
-            path: [...path, 'properties', i],
+            path: [...path, schema.allOf ? 'allOf' : 'properties', i],
```

Comment:
I analyzed the whole flow now, and having `properties` here is not intended at all.
```suggestion
            path: [...path, getCombiner(node), i],
```